php-worker/Dockerfile

```diff
@@ -30,12 +30,12 @@ RUN apk --update add wget \
   autoconf \
   cyrus-sasl-dev \
   libgsasl-dev \
+  libxslt-dev \
```

Comment:
do we use libxslt?

Reply:
@denis-chmel I'm not sure, but this was added when we were migrating to php7.4. Do you think that this will lead to any problems?